### PR TITLE
vimc-6927 update to new api

### PR DESCRIPTION
**Pull request by hillalex**
new response schema and url prefix

### CODE REVIEW

File: scripts/run-dependencies.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ docker-compose pull
 docker-compose --project-name montagu up -d
 
 # Install orderly-web
+pip3 install constellation
```
Comment (EmmaLRussell) [inline scripts/run-dependencies.sh:14]: unrelated change?

**Review by EmmaLRussell — APPROVED** [review]

File: scripts/run-dependencies.sh (hunk 1 above)
Comment (hillalex) [reply scripts/run-dependencies.sh:14]: oh yeah, python packaging being crap, shouldn't need this but seem to
